Buildings/Experimental/DHC/Loads/BaseClasses/Examples/BaseClasses/BuildingTimeSeries.mo

```diff
@@ -18,12 +18,16 @@ model BuildingTimeSeries
     annotation (Evaluate=true, Dialog(group="Configuration"));
   parameter String filNam
     "File name with thermal loads as time series";
-  parameter Real facMulHea=1
+  parameter Real facMulHea=QHea_flow_nominal /
+    (5e3 * abs(T_aLoaHea_nominal - T_aHeaWat_nominal) / 20 *
+     mLoaHea_flow_nominal / 1)
     "Heating terminal unit multiplier factor"
-    annotation(Dialog(enable=have_heaWat, group="Scaling"));
-  parameter Real facMulCoo=1
+    annotation(Dialog(enable=have_heaWat, group="Scaling", tab="Advanced"));
+  parameter Real facMulCoo=QCoo_flow_nominal /
+    (-3e3 * abs(T_aLoaCoo_nominal - T_aChiWat_nominal) / 15 *
```

Comment:
This is a similar problem as reported in https://github.com/lbl-srg/modelica-buildings/pull/3093#issuecomment-1384480665
These factors should be removed or explained in the `info` section and introduced as parameters so we know what they represent.

Reply:
I am working on edits also with input from @AntoineGautier.

Reply:
@mwetter Can you check again and see if the latest changes are satisfactory?